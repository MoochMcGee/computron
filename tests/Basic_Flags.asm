[bits 16]

clc
stc
cmc
cmc
cmc
cld
std
cli
sti

db 0xf1
