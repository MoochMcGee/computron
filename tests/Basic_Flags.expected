1000:00000000 F8 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000001 F9 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000002 F5 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=1 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000003 F5 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000004 F5 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=1 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000005 FC EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000006 FD EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000007 FA EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=1 O=0 A16 O16 X16
1000:00000008 FB EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=0 D=1 O=0 A16 O16 X16
1000:00000009 F1 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00001000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=1000 ES=0000 SS=9000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=1 O=0 A16 O16 X16
